src/js/view/floating/floating-controller.ts

```diff
@@ -237,13 +245,20 @@ export default class FloatingController {
         }
 
         this.disableFloatingUI();
-
-        // Perform resize and trigger "float" event responsively to prevent layout thrashing
-        this._model.trigger('forceResponsiveListener', {});
```

Comment:
it's important that we not trigger this when the player is not displayed. This should only be called after the transition is complete.